### PR TITLE
NSA-2508

### PR DESCRIPTION
**Pull request by jovin-joy**
NSA-2508 Support - Filter on Orgs Page

### CODE REVIEW

File: src/infrastructure/organisations/api.js [hunk 1]
```diff
@@ -160,41 +161,8 @@ const listInvitationServices = async (page, pageSize, correlationId) => {
...
-    uri += `&filterstatus=0&filterstatus=2&filterstatus=3`
-  }
-  return callOrganisationsApi(uri, 'GET', undefined, correlationId);
-};
-
-const getRequestById = async (requestId, correlationId) => {
-  return callOrganisationsApi(`organisations/requests/${requestId}`, 'GET', undefined, correlationId);
-};
-
-const updateRequestById = async (requestId, status, actionedBy, actionedReason, actionedAt, correlationId) => {
```
Comment (bridgesr) [inline src/infrastructure/organisations/api.js]: Hi @jovin-joy why have you removed these two?

Comment (jovin-joy) [reply src/infrastructure/organisations/api.js]: Hi Rhys, merge conflict confusion, I will correct it ;)

File: src/infrastructure/organisations/api.js [hunk 2]
```diff
@@ -220,8 +225,5 @@ module.exports = {
   deleteUserOrganisation,
   deleteInvitationOrganisation,
   getUserOrganisationsV2,
-  listRequests,
-  getRequestById,
```
Comment (bridgesr) [inline src/infrastructure/organisations/api.js:224]: You need to add these back as well

Comment (jovin-joy) [reply src/infrastructure/organisations/api.js:224]: Done

**Review by bridgesr — APPROVED** [review]

**Pull request merged**